docs/api/QUICKSTART.md

```diff
@@ -249,21 +248,27 @@ static void private_button_on_press(void)
         uvisor_ctx->index = 0;
 
         /* For debug purposes: Print the content of the buffer. */
-        printf("Thread count between button presses: ");
+        uvisor_ctx->pc->printf("Thread count between button presses: ");
```

Comment:
This runs in an ISR, so we shouldn't do this.

I tried adding mbed-events from a uVisor box, but all hell breaks loose. Random crashes etc.